external/jni/Android-nnstreamer-prebuilt.mk

```diff
@@ -54,12 +49,7 @@ include $(PREBUILT_SHARED_LIBRARY)
 # tensorflow-lite
 #------------------------------------------------------
 ifeq ($(ENABLE_TF_LITE),true)
-include $(CLEAR_VARS)
```

Comment:
tf-lite block is unnecessary. let's consider to remove all (line 53~63).

Reply:
Okay. I removed them!